mk/config.mk

```diff
@@ -574,6 +574,17 @@ $(eval $(call cfg-depends-all,CFG_SYSTEM_PTA,CFG_WITH_USER_TA))
 # world OS.
 CFG_DEVICE_ENUM_PTA ?= y
 
+# The attestation pseudo TA provides an interface to request a measurement of
+# a given TA binary or shared library given its UUID.
+CFG_ATTESTATION_PTA ?= n
+$(eval $(call cfg-depends-all,CFG_ATTESTATION_PTA,_CFG_WITH_SECURE_STORAGE))
+
+# RSA key size (in bits) for the attestation PTA. Must be at least 2048 which
+# is the size of the data to be signed (an SHA-256 hash).
```

Comment:
> Must be at least 2048 which is the size of the data to be signed (an SHA-256 hash).

Isn't size of SHA-256 hash 256 bits? Although I agree that choosing 2048 as RSA key size is better default.

Reply:
Actually what I wrote is complete BS :open_mouth: :smile: I will update the text and explain the key size constraint more precisely.